packages/frontend/src/features/sqlRunner/components/ContentPanel.tsx

```diff
@@ -17,13 +16,18 @@ import { notifications } from '@mantine/notifications';
 import { IconChartHistogram, IconCodeCircle } from '@tabler/icons-react';
 import {
     useCallback,
-    useDeferredValue,
     useEffect,
     useMemo,
+    useRef,
     useState,
     type FC,
 } from 'react';
-import { ResizableBox } from 'react-resizable';
+import {
+    Panel,
+    PanelGroup,
+    PanelResizeHandle,
+    type ImperativePanelHandle,
+} from 'react-resizable-panels';
```

Comment:
replaced with `react-resizable-panels` - much faster and better API (used in Semantic viewer) will replace it as well on the left sidebar